VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -66,11 +66,23 @@ def __init__(self,patching, hutil, logger,para_parser):
             self.logger.log(error_msg, True ,'Error')
             self.output_lsblk = {}
         self.devicesToInclude = [] #partitions to be included
-        self.isAnyDiskExcluded = para_parser.includedDisks[CommonVariables.isAnyDiskExcluded]
-        self.includedLunList = para_parser.includeLunList
         self.device_mount_points = []
+        self.isAnyDiskExcluded = False
+        try:
+            if(para_parser.includedDisks != None and para_parser.includedDisks != ''):
+                self.isAnyDiskExcluded = para_parser.includedDisks[CommonVariables.isAnyDiskExcluded]
```

Comment:
if the key is not present, do we expect an exception? In any case, we can probably put a check if they key exists and the value is not None before assigning to isAnyDiskExcluded.